my_main.py

```diff
@@ -0,0 +1,67 @@
+# HMMMM what should i do
```

Comment:
Remove superfluous comments.
If there is documentation at the top of the file, usually it is contained in a triple quote rather than with hashtags. For example:

```
"""
This is a script for outputting all possible additions of 7.
"""
```